release/titus-isolate.service

```diff
@@ -10,7 +10,6 @@ ExecStartPre=/usr/share/python/titus-isolate/bin/pip3 install 'netflix-spectator
 ExecStart=/usr/share/python/titus-isolate/bin/titus-isolate --admin-port 7500
 KillMode=mixed
 Restart=on-failure
-RestartSec=5
```

Comment:
Wouldn't you want to change this explicitly to be like 1s?

Reply:
I want to go with what systemd thinks is the fastest and best.